charts/aether/values.yaml

```diff
@@ -14,7 +14,9 @@ image:
 # this secret will be volumed under `/etc/secrets`
 secretName: ""
 
-config: {}
+config:
+  providersConfig:
+    scrapingInterval: 5m
```

Comment:
I don't think we should set this here, its already defaults in the code: https://github.com/re-cinq/aether/blob/main/pkg/config/config.go#L45